spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -165,13 +165,23 @@
           TableProperties.PARQUET_BATCH_SIZE, TableProperties.PARQUET_BATCH_SIZE_DEFAULT));
   }
 
+  private Schema getTableSchema() {
```

Comment:
I think this method name could be more descriptive. The verb "get" doesn't add anything so we don't normally use it. And this is also not the table schema, it is a snapshot's schema. How about renaming this to `snapshotSchema()`?

Reply:
Renamed to `snapshotSchema` as you suggest.